config/config.json

```diff
@@ -0,0 +1,24 @@
+{
+  "development": {
+    "username": "charlie",
+    "password": "",
+    "database": "charlie_bank",
+    "host": "localhost",
+    "dialect": "mysql",
+    "port": "8889"
+  },
+  "test": {
```

Comment:
Do you really need a test for this use case?

Reply:
I have removed the test variables